GAE/src/org/waterforpeople/mapping/dataexport/GraphicalSurveySummaryExporter.java

```diff
@@ -424,56 +436,93 @@ public int compare(String o1, String o2) {
                 }
 
                 FileOutputStream fileOut = new FileOutputStream(fileName);
-                wb.setActiveSheet(isFullReport ? 1 : 0);
+                wb.setActiveSheet(isFullReport ? wb.getNumberOfSheets()-1 : 0);
                 wb.write(fileOut);
                 fileOut.close();
 
             } else {
                 log.info("No questions for survey: "
                         + criteria.get(SurveyRestRequest.SURVEY_ID_PARAM)
-                        + " - instance: " + serverBase);
+                        + " - instance: " + serverBaseUrl);
             }
         } catch (Exception e) {
             log.error("Error generating report: " + e.getMessage(), e);
         }
+        //TODO: we could send success/fail status to server for the notification tab
```

Comment:
Can you make a note of this under the https://github.com/akvo/akvo-product-design/issues/218 repo and remove the comment?